README.md

```diff
@@ -537,6 +537,23 @@ https://github.com/gtri/lowendinsight
 https://github.com/gtri/lowendinsight-get
 ```
 
+### GitHub Action
```

Comment:
Can you add a note, that this only works for "elixir" projects right now.  But more support coming?